check_test.go

```diff
@@ -152,6 +153,37 @@ func doTestCheckerCheckFunc(t *testing.T, updateInterval time.Duration, err erro
 	}
 }
 
+func doTestCheckerCheckWithIgnoredFunc(t *testing.T, updateInterval time.Duration, err error, ignoreTwo bool, expectedStatus AvailabilityStatus) {
```

Comment:
If we only have one test in which this function is used, we can remove it and move the test code directly into the test function itself (`TestWhenOneIgnoredCheckFailedThenAggregatedResultUp`).